### PR TITLE
element utils

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

**Comment by ngokevin** [comment]
@mstriemer is this something we want, or is the approach okay?

File: core/element_utils.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        // Highlight new active nodes based on current page.
+        var activeLinks = element.querySelectorAll(
+            'a[href="' + (path || window.location.pathname) + '"]');
+        for (i = 0; activeLinks && (i < activeLinks.length); i++) {
+            activeLinks[i].classList.add(activeNodeClass);
+        }
+    }
+
+    return {
+        nlForEach: nlForEach,
```
Comment (mstriemer) [inline core/element_utils.js]: I think I'd rather this just be `forEach` or `each`.

**Comment by mstriemer** [comment]
It might make sense to put this in marketplace-elements instead under `elements/utils`. However that currently isn't feasible since that module isn't broken up.

**Comment by ngokevin** [comment]
I think ```marketplace-elements``` should consume mcm. I don't see marketplace-elements as the central development area for Custom Elements, but as a place to distribute elements that are expected to be reused (so far all are unused or Fireplace-only, so there's not much use case). Putting this in mcm would make it easier for projects to write their own Custom Elements.

**Comment by mstriemer** [comment]
Fair enough. The banners were used on the devhub before but I don't think that's used anymore.

**Pull request merged**